### PR TITLE
switched to ISO date format on issue finder

### LINKED ISSUE #517: Use ISO-8601 date format on the issue finder
## Description
Currently the issue finder formats dates as per the clients' locales.

https://github.com/creativecommons/creativecommons.github.io-source/blob/bd8fc4db7f2035e1009bc5502bd3b120ea8e6ce2/webpack/js/components.js#L72

This localisation is not immediately apparent and can cause confusion. Switching to the [ISO-8601 format](https://en.wikipedia.org/wiki/ISO_8601) for the dates should eliminate said confusion.

## Reproduction

1. Visit the [Issue Finder](https://opensource.creativecommons.org/contributing-code/issue-finder/).
2. See the non-standard dates.

## Expectation
The date should follow the ISO standard.

## Resolution

- [ ] I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by ooyeraju**
## Fixes

Fixes #517  by @dhruvkb 

## Description
Used ISO-8601 date format on the issue finder as mentioned in issue

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: webpack/js/components.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ export const IssueCard = {
   computed: {
     dateCreated() {
       const dateCreated = new Date(this.issue.createdAt * 1000)
-      return dateCreated.toLocaleDateString()
+      return dateCreated.toISOString()
```
Comment (dhruvkb) [inline webpack/js/components.js:74]: I believe this method would return a string like `'2020-09-30T00:00:00.000Z'` whereas we would preferably want just the date component `'2020-09-30'`.

**Review by dhruvkb — CHANGES_REQUESTED** [review]

File: webpack/js/components.js (hunk 1 above)
Comment (ooyeraju) [reply webpack/js/components.js:74]: hey, @dhruvkb  should I return a substring here, by formatting the isoString?

Comment (dhruvkb) [reply webpack/js/components.js:74]: Yeah I think that should be fine.

File: webpack/js/components.js [hunk 2]
```diff
@@ -71,7 +71,7 @@ export const IssueCard = {
   computed: {
     dateCreated() {
       const dateCreated = new Date(this.issue.createdAt * 1000)
-      return dateCreated.toISOString()
+      return dateCreated.toISOString().substring(0,10)
```
Comment (dhruvkb) [inline webpack/js/components.js:74]: Instead of hardcoding `10`, can we not do `.indexOf('T')`? That seem's cleaner imo.

**Review by dhruvkb — CHANGES_REQUESTED** [review]

File: webpack/js/components.js (hunk 2 above)
Comment (ooyeraju) [reply webpack/js/components.js:74]: Ok, can I use split instead of substring as it will be much better?

Comment (dhruvkb) [reply webpack/js/components.js:74]: Whichever you prefer.

**Review by dhruvkb — APPROVED** [review]

**Comment by dhruvkb** [comment]
Congratulations @ooyeraju!

**Pull request merged**